jupyterhub/proxy.py

```diff
@@ -311,7 +311,9 @@ def validate_routespec(self, routespec):
                                 spec, route['target'], spawner.server,
                             )
                             futures.append(self.add_user(user, name))
-                elif spawner._spawn_pending:
+                elif spawner.pending:
+                    # don't consider routes for spawners in any pending event stale
```

Comment:
stale -> state

Reply:
I actually meant 'stale', but I see how it's unclear. Will rephrase.